inc/tag.class.php

```diff
@@ -539,6 +539,43 @@ static function preKanbanContent($params = []) {
       return null;
    }
 
+   public static function kanbanItemMetadata($params) {
+      global $DB;
+
+      if (!Session::haveRight(PluginTagTag::$rightname, READ)) {
+         return $params;
+      }
+
+      if (isset($params['itemtype']) && isset($params['items_id'])) {
+         $iterator = $DB->request([
+            'SELECT'    => [
+               'name',
+               'comment',
+               'color'
+            ],
+            'FROM'      => PluginTagTagItem::getTable(),
+            'LEFT JOIN' => [
+               PluginTagTag::getTable() => [
+                  'FKEY'   => [
+                     PluginTagTag::getTable()      => 'id',
+                     PluginTagTagItem::getTable()  => 'plugin_tag_tags_id'
+                  ]
+               ]
+            ],
+            'WHERE'     => [
+               'itemtype'  => $params['itemtype'],
+               'items_id'  => $params['items_id']
+            ]
+         ]);
+
+         $params['metadata']['tags'] = $params['metadata']['tags'] ?? [];
+         foreach ($iterator as $data) {
+            $params['metadata']['tags'][] = htmlentities($data['name']);
```

Comment:
Data stored in DB should already be encoded. If it is not, then the fix should be done in the logic that creates the tag.
```suggestion
            $params['metadata']['tags'][] = $data['name'];
```